samza-api/src/main/java/org/apache/samza/operators/MessageStream.java

```diff
@@ -82,17 +82,19 @@
    * non-stream systems (e.g., an external database).
    *
    * @param sinkFn the function to send messages in this stream to an external system
+   * @return this {@link MessageStream} object
```

Comment:
This is not as detailed as I would need if I were the user. 

Is "this" stream before or after the external stream? Are there any concerns that should be taken into account when using it for chaining? What if I want to consume the external stream, say, for modularity/testing purposes?

Same comment for the return param of sink() below.